src/components/CaptureDetailDialog.js

```diff
@@ -169,6 +208,15 @@ function CaptureDetailDialog(props) {
             </div>
           )}
         </Grid>
+        <Divider />
+        <Grid item>
+          <Typography className={classes.subtitle}>Capture Token</Typography>
+          <Typography variant="body1">
+            {capture.token != null
```

Comment:
Assuming the standard LoopBack pattern is followed, this will be `capture.tokenId` (from the `token_id` column in the `trees` table).
It might be best to check for `null` and `undefined` separately:
- `capture.tokenId === undefined` => Impact token status unknown
- `capture.tokenId === null` => Impact token not issued

That will prevent misleading information if this is released before the API change is made.